packages/manager/modules/pci/src/projects/project/storages/databases/database/pools/information/index.scss

```diff
@@ -0,0 +1,8 @@
+.information {
```

Comment:
**nitpick(blocking):** We tend to avoid using custom styles. Can't this be handled via helpers?